Changelog.rst

```diff
@@ -39,7 +39,7 @@ Breaking Changes Since 6.0.0-0
 Upgrade Issues
 """"""""""""""
 
-* You **MUST** read the ``User Guide: Upgrading SIMP`` section of the
+* You **MUST** read the :ref:`upgrade-6.0.0-to-6.1.0` section of the
```

Comment:
Trevor when I look this in the View section it just shows up exactly like this:
:ref:`upgrade-6.0.0-to-6.1.0`
Not as link.

Reply:
agreed - it shows up at text in generated pdf or html, i just haven't checked it on actual RTD